sw/host/opentitanlib/src/otp/mod.rs

```diff
@@ -16,15 +16,21 @@ mod tests {
     use anyhow::Result;
     use std::path::Path;
 
+    pub(crate) fn testdata(s: &str) -> String {
+        let mut result = "sw/host/opentitanlib/src/otp/testdata/".to_string();
```

Comment:
Would a PathBuf be more appropriate?

Reply:
It looks like `sw/host/opentitanlib/src/util/image.rs` also uses a very similar function. Created #10545 to merge this now.